tests/data/test_defaults.py

```diff
@@ -38,7 +38,7 @@
     "comment_count": 0,
     "creator_id": "0",
     "created_at": "2019-01-02T21:00:30.00000Z",
-    "url": "https://todoist.com/showTask?id=2995104339",
```

Comment:
Why this change?

Reply:
It was incorrect, as the ID of the test task is `1234`:

https://github.com/Doist/todoist-api-python/blob/2fc2cf05b306e935df09605c9a8745df8fb200c6/tests/data/test_defaults.py#L25